### ISSUE TITLE
Make/Compilation Error

### ISSUE THREAD

**Issue by aramir** [modcluster/mod_proxy_cluster#82]
On `make`,  geting:
```
[ 42%] Building C object mod_manager/CMakeFiles/mod_manager.dir/mod_manager.c.o
/opt/mod_cluster/mod_proxy_cluster-main/native/mod_manager/mod_manager.c: In function process_dump
/opt/mod_cluster/mod_proxy_cluster-main/native/mod_manager/mod_manager.c:1612:13: error: a label can only be part of a statement and a declaration is not a statement
             char *status;
             ^~~~
```
for this code block:
```c
case TEXT_XML:
    char *status;
    status = "REMOVED";
    switch (ou->status) {
    case ENABLED:
	status = "ENABLED";
	break;
    case DISABLED:
	status = "DISABLED";
	break;
    case STOPPED:
	status = "STOPPED";
	break;
    }
    ap_rprintf(r, "<Context id=\"%d\" path=\"%.*s\">\
		    <Vhost>%d</Vhost>\
		    <Node>%d</Node>\
		    <Status id=\"%d\">%s</Status>\
		   </Context>",
	       id[i], (int)sizeof(ou->context), ou->context, ou->vhost, ou->node, ou->status, status);
    break;
```
surrounding the case with `{}` fixed it

**Comment by jajik** [comment]
Hello, could you please share which compiler and which flags were used?

**Comment by jajik** [comment]
Great, thank you! The fix is ready in PR #84 along with a tweak for CI jobs.

The described error manifests with gcc older than 11.1 and with all the versions of clang as you can see [here](https://godbolt.org/#z:OYLghAFBqd5QCxAYwPYBMCmBRdBLAF1QCcAaPECAMzwBtMA7AQwFtMQByARg9KtQYEAysib0QXACx8BBAKoBnTAAUAHpwAMvAFYTStJg1AB9U8lJL6yAngGVG6AMKpaAVxYM9DgDJ4GmADl3ACNMYhAAJgBmUgAHVAVCWwZnNw89eMSbAV9/IJZQ8OiLTCtshiECJmICVPdPLhKy5MrqglzAkLDImIUqmrr0xr62jvzCnoBKC1RXYmR2DgBSaL9kNywAaiWoxz78VAA6BB3sJY0AQRWotY3Mbd392jxg49Pzq8u/Ak2WJj8IJNtgB2ABCH02kM232hDwAIptJBEduDLhCoQoAO6EZAITYQPBApZg9FQqGiJSIiIgUlkukw1TwzZcDQo2l0jHYgi4/GEkGoi4coXkpiUpE0y7CqVQxk7BGsqIC6XS4LETBMADWbMlyrJWCoTFctAIEsFuqFsqiCKR2rN5rpqvVWsV7KFxLhro5sWI3yoEBWEQo2wiAFZ0KRNupg2GliHHAwAxG8BHVJNbWT3R9aWqCHMGJsFQLM5cONNaJwQ7xPBwtKRUJxHJsFLN5vdrjxSCaa6XphqQFFJIcABwANg0GhDGiHAE4J0OQ6H9JxJFXNLx6xxeAoQBpO2vpnBYEg0CxYnQwuRKCez/RwshgFxH6QsAA3PALABqeEwmIA8rFGE4DsaGNMJtwgYI11IYI/GqABPIDeBg5hiDg39gm0TBrEQ0gTzYQRfwYWgEO7Z9MD%2BIxxFI/A1WsPAX0wbdSMwVQsNcAhFlrb5Sig55VXg5wsCgggfRYRDpioAxgAUL8f3/QDuF4fhBBEMR2CkGRBEUFR1FI3RGgMIwQFMYxzD47dIGmVBYnKJj1wY4gfSwCzASaLDynsBgnBceovC8sYunCRpMiSARBgaOIElChgAoKbphlKdyWn6WofKGNy6IEVoaliiZhhS8K9BGHK/E6OKgumZs5gWCQywrVdSI3SNRwAWhHSRNmAZBkGZLhDi4fFcEIEhgyiLhJl4LstEmXt%2Bz6ocNBHEcuBDYFgTHIcpAiEclw4FdSGrWsNy3Hc927A8YEQEBZgIWJ2MvCBr3PYgAlYRZVFa9rOu65kIkOCJeEwfAiCcvRlOEURxA08HtLUKD9NITFiCYWJxN2ysDqgjdf3Yu6flQKhmpHNqOq6nrH36/FnFPZ7RvGyb91mqIokOZm2fZjndv2w7104E7dymntdoBzHGr5s7pumBzEjsSQgA%3D).
